docs/dashboard/layout.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: docs-dashboard
+toc: toc-dashboard.html
+title: Layouting Dashboard
+---
+
+<!-- This document contains original Japanese description in comments -->
+
+### Basic UI Elements of Dashboard
+
+<!-- ### Dashboard画面の基本部品 -->
+
+<!--　Node-RED Dashboardは`Widget`、`Group`、および、`Tab`から構成されます。WidgetはButton、ChartなどGUI部品です。TabはWebブラウザ上に表示するダッシュボード画面全体を表します。Tabは複数定義でき、サイドメニューからの選択、もしくは、ui_controlノードで切り替えて一つのタブを表示できます。Groupは複数のWidgetを配置する領域を定義するために使用します。GroupおよびTabはConfigノードです。-->
+
+  The Node-RED Dashboard consists of `Widget`,`Group`, and `Tab`.  Widget is a GUI element such as button or chart.  Tab represents the entire dashboard screen displayed on the Web browser.  Multiple Tabs can be defined, and one tab can be displayed by selecting from the side menu or switched with the `ui_control node`.  Group is used to define the area where multiple widgets will be placed.  Group and Tab are defined as configuration nodes.
+
+### Layouting Dashboard Elements
+
+<!-- ### 画面要素のレイアウト -->
+
+<img src="images/layout/Site.png" style="float: right; margin-left: 20px"/>
+
+<!--　ダッシュボードは一定の大きさのグリッドを基本要素とし、Widgetは高さと幅、Groupは幅をグリッド数を単位として定義します。グリッドの単位要素の大きさはdashboardサイドバーのSiteタブで設定できます。-->
+
+  The dashboard layout is defined using a grid with fixed size element.  The widget has the height and width, and the group has the width in units of the number of grid elements. The size in pixels of the grid's unit element can be set by '*1x1 Widget Size*' of the `Site` tab of the dashboard sidebar.
+
+<!-- SIteタブでは、Widget間の間隔およびおよびグループ間の間隔と余白も設定できます。 -->
+
+On the `Site` tab, spacing between widgets and the spacing and paddings between groups can also be specified.
+
+<br style="clear:both"/>
+
+<img src="images/layout/Layout.png" style="float:left; margin-right: 20px"/>
+
+<!--　WidgetはGroup内に、GroupはTab内に、それぞれの幅に収まるよう配置を調整します。これら要素の階層関係は、dashboardサイドバーのLayoutタブで確認できます。-->
+
+  The widgets are arranged in the group and the group in the tab so that they fit within their respective widths.  The hierarchical relationship of these elements can be managed in the `Layout` tab of the dashboard sidebar.
+
+<!--　Dashboardのレイアウト処理では、次のように画面のレイアウトを行います。
+
+1. 各グループのWidgetについて、上から下、左から右にGroupに配置する。
+
+2. 各グループについて、上から下、左から右にTabに配置する。-->
+
+While layouting the dashboard screen, placements of screen elements are determined as follows:
```

Comment:
-> When laying out the dashboard screen, the placement of screen elements is ....